ui/components/app/add-network/index.scss

```
@@ -8,10 +8,13 @@
     }
   }
 
-  &__token-image {
-    margin-right: 7px;
-    height: 24px;
-    width: 24px;
+  &__warning-tooltip {
+    color: var(--ui-4);
```

The colors here need to be updated to use the new design system color tokens @georgewrmarshall can probably assist in recommendations.

---

Nice catch! Yes, this can be updated to new design system color token css variable
```suggestion
    color: var(--color-text-alternative);
```

---

Done